README.md

```diff
@@ -20,6 +20,12 @@ CephFS CSI plugins, see documentation in `docs/`.
 
 For example usage of RBD and CephFS CSI plugins, see examples in `examples/`.
 
+## Contributing to this repo
+
+Please follow [contibution guide](https://github.com/ceph/ceph-csi/tree/master/docs/coding.md)
```

Comment:
I would not call the coding.md a contribution guide, I would call it the coding style guidelines, and hence reorder the links in this statement, with dev-guide first and then coding guidelines next.